src/main/java/com/ncc/aif/SeedlingOntologyMapper.java

```diff
@@ -0,0 +1,401 @@
+                    "Manufacture.Artifact",
+                    "Movement.TransportArtifact", "Movement.TransportPerson",
+                    "Personnel.Elect", "Personnel.EndPosition", "Personnel.Nominate", "Personnel.StartPosition",
+                    "Transaction.Transaction", "Transaction.TransferControl", "Transaction.TransferMoney",
+                    "Transaction.TransferOwnership"), it -> it);
+
+    private final ImmutableList<String> SEEDLING_EVENT_TYPES = ImmutableList.of(
+            // those in the first block match the seedling ontology except...
+            "CONFLICT_ATTACK", "CONFLICT_DEMONSTRATE",
+            "CONTACT_BROADCAST", "CONTACT_CONTACT", "CONTACT_CORRESPONDENCE", "CONTACT_MEET",
+            "JUSTICE_ARREST-JAIL",
+            "LIFE_DIE", "LIFE_INJURE", "MANUFACTURE_ARTIFACT",
+            "MOVEMENT_TRANSPORT-ARTIFACT",
+            "MOVEMENT_TRANSPORT-PERSON", "PERSONNEL_ELECT",
+            "PERSONNEL_START-POSITION", "TRANSACTION_TRANSACTION", "TRANSACTION_TRANSFER-MONEY",
+            "TRANSACTION_TRANSFER-OWNERSHIP",
+            "BUSINESS_DECLARE-BANKRUPTCY",
+            "JUSTICE_ACQUIT",
+            "JUSTICE_APPEAL", "JUSTICE_CHARGE-INDICT", "JUSTICE_CONVICT", "JUSTICE_EXECUTE",
+            "JUSTICE_EXTRADITE", "JUSTICE_FINE", "JUSTICE_RELEASE-PAROLE", "JUSTICE_SENTENCE",
+            "JUSTICE_SUE", "JUSTICE_TRIAL-HEARING", "LIFE_BE-BORN", "LIFE_MARRY", "LIFE_DIVORCE",
+            "PERSONNEL_NOMINATE", "PERSONNEL_ELECT", "BUSINESS_END-BUSINESS",
+            "BUSINESS_START-BUSINESS", "BUSINESS_MERGE", "CONTACT_CORRESPONDENCE",
+            "PERSONNEL_END-POSITION");
+
+    private final ImmutableList<String> SEEDLING_EVENT_TYPES_WITH_DOTS =
+            ImmutableList.copyOf(Lists.transform(SEEDLING_EVENT_TYPES, it -> it.replace('_', '.')));
+
+    /* Unused intermediate steps, useful for debugging
+    private final ImmutableMap<String, String> SEEDLING_EVENT_TYPES_MAP =
+            Maps.toMap(SEEDLING_EVENT_TYPES, it -> it);
+
+    private final ImmutableMap<String, String> SEEDLING_EVENT_TYPES_WITH_DOTS_MAP =
+            Maps.toMap(SEEDLING_EVENT_TYPES_WITH_DOTS, it -> it.replace('.', '_'));
+*/
+
+    private final ImmutableMap<String, String> EVENT_TYPES_STRMAP =
+            ImmutableMap.<String, String>builder()
+                    .putAll(Maps.toMap(SEEDLING_EVENT_TYPES, it -> it))
+                    // or seedling types with dots instead of underscores (more ACE-like)
+                    .putAll(Maps.toMap(SEEDLING_EVENT_TYPES_WITH_DOTS, it -> it.replace('.', '_')))
+                    .putAll(SEEDLING_EVENT_TYPES_NIST)
+                    // or these remaining special cases
+                    .put("BUSINESS.END-ORG", "BUSINESS_END")
+                    .put("BUSINESS.MERGE-ORG", "BUSINESS_MERGE")
+                    // needed to read RPI Seedling output
+                    .put("CONTACT.PHONE-WRITE", "CONTACT_CORRESPONDENCE")
+                    .build();
```

Comment:
I didn't see that it was part of SEEDLING. Marking resolved

Reply:
"PERSONNEL.END-POSITION" to "PERSONNEL_END-POSITION" is removed because it's already added to EVENT_TYPES_STRMAP because PERSONNEL_END-POSITION is in SEEDLING_EVENT_TYPES, so therefore PERSONNEL.END-POSITION gets added via SEEDLING_EVENT_TYPES_WITH_DOTS.  I initially got a "duplicate key" error.